packages/perseus/src/question-paragraph.tsx

```diff
@@ -16,7 +17,7 @@ class QuestionParagraph extends React.Component<Props> {
         // to attach some metadata to top-level QuestionParagraphs:
         return (
             <div
-                className={className}
+                className={this.props.inline ? undefined : className}
```

Comment:
**issue**: We will lose the `className` provided in a prop with this approach.  I _think_ we need to split out that prop from the local `className` so we can prevent applying the `"paragraph"` class but still respect any provided. `props.className`